deps.windows/70-vpl.ps1

```diff
@@ -0,0 +1,87 @@
+param(
+    [string] $Name = 'vpl',
+    [string] $Version = 'v2023.3.0',
+    [string] $Uri = 'https://github.com/oneapi-src/oneVPL.git',
+    [string] $Hash = 'e12ace9761bb52786409e830f619916b86e87fc5'
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Clean {
+    Set-Location $Path
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+    $Options = @(
```

Comment:
Should probably use the following instead:

```
    if ( $ForceStatic -and $script:Shared ) {
        $Shared = $false
    } else {
        $Shared = $script:Shared.isPresent
    }    

    $OnOff = @('OFF', 'ON')
    $Options = @(
        $CmakeOptions
[...]
        "-DBUILD_SHARED_LIBS:BOOL=$($OnOff[$Shared])"
    )
```

We probably might simplify this across the board later, but for now that's the common pattern.

Reply:
I've added this via force push.